src/main/java/kitchenpos/order/applicaiton/OrderService.java

```diff
@@ -0,0 +1,55 @@
+package kitchenpos.order.applicaiton;
+
+import kitchenpos.menu.domain.Menu;
+import kitchenpos.menu.domain.MenuRepository;
+import kitchenpos.order.domain.*;
+import kitchenpos.order.dto.OrderRequest;
+import kitchenpos.order.dto.OrderResponse;
+import kitchenpos.order.dto.OrderStatusRequest;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class OrderService {
+    private final MenuRepository menuRepository;
+    private final OrderRepository orderRepository;
+    private final OrderTableService orderTableService;
+
+    public OrderService(
+            final MenuRepository menuRepository,
+            final OrderRepository orderRepository,
+            final OrderTableService orderTableService) {
+        this.menuRepository = menuRepository;
+        this.orderRepository = orderRepository;
+        this.orderTableService = orderTableService;
+    }
+
+    @Transactional
+    public OrderResponse create(final OrderRequest orderRequest) {
+        OrderTable orderTable = orderTableService.findOrderTable(orderRequest.getOrderTableId());
+        List<Menu> menus = menuRepository.findAllById(orderRequest.findAllMenuIds());
+        Order order = orderRequest.toOrder(orderTable, OrderStatus.COOKING, menus);
+        order.checkValidOrder(menus.size());
+
+        return OrderResponse.of(orderRepository.save(order));
+    }
+
+    public List<OrderResponse> list() {
+        return orderRepository.findAll()
+                .stream().map(OrderResponse::of)
+                .collect(Collectors.toList());
+    }
+
+    @Transactional
+    public OrderResponse changeOrderStatus(final Long orderId, final OrderStatusRequest orderStatusRequest) {
+        final Order savedOrder = orderRepository.findById(orderId)
+                .orElseThrow(IllegalArgumentException::new);
+
+        savedOrder.throwIfCompleted();
+        savedOrder.changeOrderStatus(orderStatusRequest.getOrderStatus());
```

Comment:
```suggestion
        savedOrder.changeOrderStatus(orderStatusRequest.getOrderStatus());
```
주문 상태를 변경할때 유효성검사를 하는건 어떨까요?